src/views/playground/Playgrounds/OrionTesting.tsx

```diff
@@ -18,15 +18,13 @@ export const OrionTesting = () => {
     },
     fetchPolicy: 'network-only',
   })
-  const [
-    getVideosConnection,
-    { data: videosConnectionData, loading: videosConnectionLoading },
-  ] = useGetVideosConnectionLazyQuery({
-    variables: {
-      first: 5,
-    },
-    fetchPolicy: 'network-only',
-  })
+  const [getVideosConnection, { data: videosConnectionData, loading: videosConnectionLoading }] =
+    useGetVideosConnectionLazyQuery({
+      variables: {
+        first: 5,
+      },
+      fetchPolicy: 'network-only',
+    })
```

Comment:
Why there are so many prettier-related changes? 
Also when I hit `ctrl + s` on these files my editor autocorrects me and brings back prev version.

https://user-images.githubusercontent.com/51168865/131966510-f11bf37a-56cf-453f-b515-0c238a1f5cd6.mov

Reply:
There are changes because I've upgraded to a newer version and those format the code differently. Your editor probably still uses some cached old version of prettier. Make sure you select the correct one

Reply:
Yeah sorry, my mistake. Restarted extension host and everything works fine now!